contentcuration/contentcuration/static/js/edit_channel/share/views.js

```diff
@@ -44,9 +47,20 @@ var ShareView = BaseViews.BaseView.extend({
     render: function() {
         this.$el.html(this.template({
             channel:this.model.toJSON(),
-            user: this.current_user.toJSON()
+            user: this.current_user.toJSON(),
```

Comment:
this.model refers to the current channel (represented as ChannelModel) whereas this.current_user has the user model (represented as UserModel)

One motivation behind keeping the models the same as the API is so that Backbone can make a simple put request with all of the data formatted to match the restful API